### PR TITLE
Edith/make website

### PR DESCRIPTION
**Pull request by ediscovery22**

### CODE REVIEW

File: make-webpage.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<html lang="en">
+    
+</html>
```
Comment (BrotherKristoph) [inline make-webpage.html]: This is all valid html :+1: also, it's common practice to name the main file of your website `index.html`, if you want this to be the first thing someone sees when they visit the site.

Comment (ediscovery22) [reply make-webpage.html]: Ahh, thank you. I will change that up. May I ask, how does one get to see if their code works, like how do I run the program?

Comment (BrotherKristoph) [reply make-webpage.html]: [This `npm` package](https://www.npmjs.com/package/http-server) will help ya set up a local webserver so you can view the html you write in your browser. You'll need `npm` installed (run `npm --version` in git bash to check) then run `npm install -g http-server`. Once you have `http-server` installed, you can run the command `http-server` in the `hirelings-ui` directory to start up a local webserver. Hit one of the URLs listed in the terminal to see yer code!

Comment (ediscovery22) [reply make-webpage.html]: This is as close as I can get to installing npm: 

Edith@Edith MINGW32 /
$ node -v
v9.1.0

Edith@Edith MINGW32 /
$ npm -v
Error: EPERM: operation not permitted, mkdir 'C:\Program Files (x86)\Git\'~'
TypeError: Cannot read property 'get' of undefined
    at errorHandler (C:\Program Files\nodejs\node_modules\npm\lib\utils\error-handler.js:205:18)
    at C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js:83:20
    at cb (C:\Program Files\nodejs\node_modules\npm\lib\npm.js:214:22)
    at C:\Program Files\nodejs\node_modules\npm\lib\npm.js:252:24
    at C:\Program Files\nodejs\node_modules\npm\lib\config\core.js:81:7
    at Array.forEach (<anonymous>)
    at C:\Program Files\nodejs\node_modules\npm\lib\config\core.js:80:13
    at f (C:\Program Files\nodejs\node_modules\npm\node_modules\once\once.js:25:25)
    at afterExtras (C:\Program Files\nodejs\node_modules\npm\lib\config\core.js:178:20)
    at C:\Program Files\nodejs\node_modules\npm\node_modules\mkdirp\index.js:35:29
C:\Program Files\nodejs\node_modules\npm\lib\utils\error-handler.js:205
  if (npm.config.get('json')) {
                 ^

TypeError: Cannot read property 'get' of undefined
    at process.errorHandler (C:\Program Files\nodejs\node_modules\npm\lib\utils\error-handler.js:205:18)
    at emitOne (events.js:125:13)
    at process.emit (events.js:221:7)
    at process._fatalException (bootstrap_node.js:382:26)
5.5.1

Comment (BrotherKristoph) [reply make-webpage.html]: OK, looks like you have node installed, so don't worry about checking npm. Go ahead and continue with the instructions :+1:

Comment (ediscovery22) [reply make-webpage.html]: Yay, thank you. I got it to work :)

**Pull request closed without merging**